doc/source/commands/keg.rst

```diff
@@ -1,28 +1,73 @@
 keg
 ===
 
+.. _keg_synopsis:
+
 SYNOPSIS
 --------
 
 .. code:: bash
 
+   keg (-r RECIPES_ROOT|--recipes-root=RECIPES_ROOT)
+       [-a ADD_DATA_ROOT] ... [-d DEST_DIR] [-fv]
+       SOURCE
+   keg -h | --help
    keg -h | --help
 
 DESCRIPTION
 -----------
 
-Description
+Keg is a tool which helps to create and manage image descriptions suitable
+for the `KIWI <https://osinside.github.io/kiwi/>`__ appliance builder. Its
+main use case is to keep control over a larger amount of image descriptions
+and prevent duplication of description data.
+
+The key component for Keg is a data structure called `image definition tree`.
+This data structure contains all information to create KIWI image
+descriptions and provides data in a way that no or as little as possible
```

Comment:
s/descriptions and ...exists./descriptions. Keg is implemented such that data inheritance is possible to reduce data duplication in the `image definition tree`./

Reply:
done